shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -47,6 +47,7 @@ - (instancetype)initWithDelegate:(id<FlutterViewEngineDelegate>)delegate opaque:
   if (self) {
     _delegate = delegate;
     self.layer.opaque = opaque;
+    self.backgroundColor = UIColor.clearColor;
```

Comment:
I would add a short comment here that explains why this color is important

Reply:
Thx for reviewing this, I will add a comment immediately^_^

Reply:
Comment added.